Gruntfile.js

```diff
@@ -2,6 +2,8 @@ module.exports = function(grunt) {
   'use strict';
   require('google-closure-compiler').grunt(grunt);
 
+  grunt.loadNpmTasks('grunt-stylelint');
```

Comment:
Could you put this with the other load tasks, or move them all up here if you feel they should be above the configs.

Reply:
I've moved it below to match the [sample Gruntfile](https://gruntjs.com/sample-gruntfile) from the Grunt website.